packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -568,6 +573,7 @@ export class KubernetesClient {
     await this.refresh();
     // notify change
     this._onDidUpdateKubeconfig.fire({ type: 'UPDATE', location });
+    this.apiSender.send('kubernetes-context-update');
```

Comment:
I noticed that too, but I wanted to keep this similar to how we use Pods + Containers for stores and rely on api sender requests.

We do not need a `onDidUpdateKubeConfig` listener as there already is one in the kube context extension.

The reason why I do not want to add a `onDidUpdateKubeConfig` is that it's handled by the kube context extension, so if we disable the context extension it would also disable us updating the settings as well since the user does not want Podman Desktop to check `.kube/config` all the time.